IntegrationTests/tests_04_performance/test_01_resources/test_schedule_10000_tasks.swift

```diff
@@ -18,17 +18,20 @@ import NIOPosix
 func run(identifier: String) {
     let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
     let loop = group.next()
-    let dg = DispatchGroup()
 
     measure(identifier: identifier) {
-        loop.execute {
-            for _ in 0..<10_000 {
-                dg.enter()
-                loop.scheduleTask(in: .nanoseconds(0)) { dg.leave() }
+        let counter = try! loop.submit { () -> Int in
+            var counter: Int = 0
+
+            for _ in 0..<10000 {
+                loop.scheduleTask(in: .hours(1)) {
+                    counter += 1
```

Comment:
A minor nit, but for benchmark tests it's often useful to elide _all_ unnecessary work.

```suggestion
                    counter &+= 1
```